src/assets/i18n/en.json5

```diff
@@ -4307,18 +4307,26 @@
 
   "relationships.Publication.isJournalIssueOfPublication.JournalIssue": "Journal Issue",
 
+  "relationships.Publication.isContributorOfPublication.Person": "Contributor",
+
+  "relationships.Publication.isContributorOfPublication.OrgUnit": "Contributor (organizational units)",
+
   "relationships.Person.isPublicationOfAuthor.Publication": "Publications",
 
   "relationships.Person.isProjectOfPerson.Project": "Research Projects",
 
   "relationships.Person.isOrgUnitOfPerson.OrgUnit": "Organizational Units",
 
+  "relationships.Person.isPublicationOfContributor.Publication": "Publications",
```

Comment:
When editing a Person Entity, you'll see *two* relationships that say "Publications".  This one, and the one above for `isPublicationOfAuthor`.  

Therefore, I'd recommend we give this a better name.  Maybe "Publications (contributor to)"?  We need to somehow differentiate these two different relationships in the user interface.